### DISCUSSION TITLE
Tyre type info?

### DISCUSSION THREAD

**Discussion by andyKien** [snipem/gt7dashboard#30] [Q&A]
Hi, I'm missing the type of tyres currently in use.
Does anyone know where in the packet this info is?
I guess it must be in there, since they show tyre info in the overlay in the World Series.

**Comment by andyKien** [comment]
And also tyre wear

**Reply by snipem** [reply]
It is not in there. There is no tyre data in the telemetry data.

They use a different software build for the World Series.
